CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## Version 2.1.0
+- Incorporate changes for App Store Server API v1.11 and App Store Server Notifications v2.11 [https://github.com/apple/app-store-server-library-java/pull/94]
+- Add a proxy authenticator support [https://github.com/apple/app-store-server-library-java/pull/93]
```

Comment:
Should this read `Add proxy authenticator support`?

Reply:
Updated